cmds/core/brctl/brctl_linux_test.go

```diff
@@ -75,12 +76,13 @@ func TestRun(t *testing.T) {
 			argv:   []string{"addif", "bridge0", "eth0", "eth1"},
 			expErr: errFewArgs,
 		},
-		{
-			name:   "delif",
-			argv:   []string{"delif", "bridge0", "eth0"},
-			expErr: unix.ENODEV,
-			skipVM: true,
-		},
+		// see https://github.com/hugelgupf/vmtest/issues/130
```

Comment:
nit: maybe let's reference the original issue here (in 130 it says it is a duplicate) to keep it as close to the underlying issue as possible? https://github.com/hugelgupf/vmtest/issues/127

Reply:
Just saw that, referenced it in my issue, as well.